src/DAMA/DoctrineTestBundle/Doctrine/DBAL/StaticDriver.php

```diff
@@ -126,11 +123,7 @@ public static function isKeepStaticConnections(): bool
     public static function beginTransaction(): void
     {
         foreach (self::$connections as $con) {
-            try {
-                $con->beginTransaction();
-            } catch (\PDOException $e) {
```

Comment:
we should not silence any possible error here.

I could not reproduce why this is needed. Some legacy left-over or so :confused: